src/cli.ts

```diff
@@ -81,9 +76,8 @@ import {
 			process.exit(1);
 		}
 
-		execSync(`git commit -m "${aiCommitMessage}"`, {
```

Comment:
This was a pretty big problem. If OpenAI returned a message with a double-quote in it, it would break.